circle.yml

```diff
@@ -18,7 +18,7 @@ test:
     - npm run test-cov
 deployment:
   npm:
-    branch: master
+    tag: /[0-9]+.*/
```

Comment:
should not match non number in the middle expect `.`, and tag should end with number.